adyenv6core/resources/adyenv6core-items.xml

```diff
@@ -293,6 +293,39 @@
                         </columntype>
                     </persistence>
                 </attribute>
+                <!-- Multibanco-->
+                <attribute qualifier="adyenMultibancoEntity" type="java.lang.String">
+                    <description>Multibanco entity</description>
+                    <persistence type="property">
+                        <columntype>
+                            <value>HYBRIS.LONG_STRING</value>
+                        </columntype>
```

Comment:
Please check all column types. if the length is not expected to be more than 255 characters, using HYBRIS.LONG_STRING is overhead. You can omit columntype if length is within 255.